recipes/autoflake/meta.yaml

```
@@ -0,0 +1,42 @@
+{% set name = "autoflake" %}
+{% set version = "1.2" %}
+{% set hash_type = "sha256" %}
+{% set hash_val = "c103e63466f11db3617167a2c68ff6a0cda35b940222920631c6eeec6b67e807" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  {{ hash_type }}: {{ hash_val }}
+
+build:
+  number: 0
```

This looks like it could be a noarch package:
```suggestion
  noarch: python
```